packages/avatars/src/types/index.ts

```diff
@@ -29,7 +29,7 @@ export interface IAvatarProps extends HTMLAttributes<HTMLElement> {
   /**
    * Provides surface color for an avatar placed on a non-default background.
    * Accepts a [color variable](/components/theme-object#colors) key (i.e.
-   * `background.primary`) to render based on light/dark mode, or any hex value.
+   * `background.primaryEmphasis`) to render based on light/dark mode, or any hex value.
```

Comment:
```suggestion
   * `background.emphasis`) to render based on light/dark mode, or any hex value.
```

[nit] because we don't want to suggest that `primaryEmphasis` should be used for anything other than primary interactive elements.